content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -93,192 +132,209 @@ product that this newly-created serial number was just assigned to.
+dates, etc. This level of detail permits more precise tracking, for example, when handling
+perishable or regulated goods.
 
-Finally, click the :guilabel:`Validate` button to deliver the products.
+To access this page, first :ref:`select a warehouse receipt or delivery order
+<inventory/product_management/incoming-outgoing-products>`. Click on the
+:icon:`oi-view-list` :guilabel:`Moves smart button` at the top of the page.
 
-.. image:: serial_numbers/serial-numbers-detailed-operations-popup.png
-   :align: center
-   :alt: Serial numbers listed in detailed operations popup.
-
-Upon validating the delivery order, a :guilabel:`Traceability` smart button appears. Click the
-:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
-includes: a :guilabel:`Reference` document, the :guilabel:`Product` being traced, the
-:guilabel:`Date`, and the :guilabel:`Lot/Serial #` assigned.
-
-The :guilabel:`Traceability Report` can also include a :guilabel:`Reference` receipt from the
-previous purchase order (PO), if any of the product quantities shared a serial number assigned
-during receipt of that specific :abbr:`PO (purchase order)`.
-
-Manage serial numbers for different operations types
-====================================================
-
-By default in Odoo, the creation of new serial numbers is only allowed upon **receiving** products
-from a purchase order. **Existing** serial numbers cannot be used. For sales orders, the opposite is
-true: new serial numbers cannot be created on the delivery order, only existing serial numbers can
-be used.
-
-To change the ability to use new (or existing) serial numbers on any operation type, go to
-:menuselection:`Inventory app --> Configuration --> Operations Types`, and select the desired
-:guilabel:`Operation Type`.
-
-For the :guilabel:`Receipts` operation type, found on the :guilabel:`Operations Types` page, the
-:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Receipts` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`, and then
-clicking the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes.
-
-For the :guilabel:`Delivery Orders` operation type, located on the :guilabel:`Operations Types`
-page, the :guilabel:`Create New Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Delivery Orders` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`,
-and clicking the checkbox beside the :guilabel:`Create New Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Be sure to click :guilabel:`Save` to save changes.
-
-.. image:: serial_numbers/serial-numbers-operations-types.png
+In the :guilabel:`Lot/Serial Number` column, manually type (or select from the dropdown)
```

Comment:
Our team is quite particular about standardizing this to "drop-down menu", with a hyphen :see_no_evil: 
```suggestion
In the :guilabel:`Lot/Serial Number` column, manually type (or select from the drop-down menu)
```